<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="1" total="8" passed="3">
  <reporter-output>
    <line>
      <![CDATA[[INFO]: The test Browser is FIREFOX !!!]]>
    </line>
    <line>
      <![CDATA[[INFO]:  The application url is :- http://s-login.cengage.com/]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: PageTitle for NG-30181-LoginPage contains: 'Cengage Learning - Login'.]]>
    </line>
    <line>
      <![CDATA[[INFO]: log into the application using credentials 'kajal.inst@qaittest.com / A111111']]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: PageTitle for NG-30181-InstructorPage contains: 'Instructor SSO - Instructor Resource Center'.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element pageTitle with text Instructor Resource Center is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element courseOperation with text Manage Course is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT PASS]: element courseLink with text Mayank Test Book-Automated is displayed.]]>
    </line>
    <line>
      <![CDATA[[ASSERT FAIL]: Element helloUser not found on the Header !!!]]>
    </line>
    <line>
      <![CDATA[[INFO]: The Test: NG_30181_TC04_INSTRUCTOR_DELETES_COURSE COMPLETED!
]]>
    </line>
  </reporter-output>
  <suite name="com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course" duration-ms="106212" started-at="2015-10-06T10:43:47Z" finished-at="2015-10-06T10:45:33Z">
    <groups>
    </groups>
    <test name="Command line test 2a15b366-bb28-4cd9-be7a-e20e75b8651d" duration-ms="106212" started-at="2015-10-06T10:43:47Z" finished-at="2015-10-06T10:45:33Z">
      <class name="com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course">
        <test-method status="PASS" signature="start_test_session(java.lang.String)[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="start_test_session" is-config="true" duration-ms="6410" started-at="2015-10-06T16:13:47Z" finished-at="2015-10-06T16:13:53Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[[INFO]: The test Browser is FIREFOX !!!]]>
            </line>
            <line>
              <![CDATA[[INFO]:  The application url is :- http://s-login.cengage.com/]]>
            </line>
          </reporter-output>
        </test-method> <!-- start_test_session -->
        <test-method status="PASS" signature="Step_01_Instructor_Logs_in_to_the_Application()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="Step_01_Instructor_Logs_in_to_the_Application" duration-ms="25377" started-at="2015-10-06T16:13:53Z" finished-at="2015-10-06T16:14:18Z">
          <reporter-output>
            <line>
              <![CDATA[[ASSERT PASS]: PageTitle for NG-30181-LoginPage contains: 'Cengage Learning - Login'.]]>
            </line>
            <line>
              <![CDATA[[INFO]: log into the application using credentials 'kajal.inst@qaittest.com / A111111']]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: PageTitle for NG-30181-InstructorPage contains: 'Instructor SSO - Instructor Resource Center'.]]>
            </line>
          </reporter-output>
        </test-method> <!-- Step_01_Instructor_Logs_in_to_the_Application -->
        <test-method status="PASS" signature="takeScreenshotonFailure(org.testng.ITestResult)[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="takeScreenshotonFailure" is-config="true" duration-ms="0" started-at="2015-10-06T16:14:18Z" finished-at="2015-10-06T16:14:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Step_01_Instructor_Logs_in_to_the_Application status=SUCCESS method=NG_30181_TC03_Instructor_Deletes_Course.Step_01_Instructor_Logs_in_to_the_Application()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] output=[ASSERT PASS]: PageTitle for NG-30181-LoginPage contains: 'Cengage Learning - Login'.]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotonFailure -->
        <test-method status="PASS" signature="Step_02_Instructor_Manage_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="Step_02_Instructor_Manage_Course" duration-ms="13224" started-at="2015-10-06T16:14:18Z" depends-on-methods="com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course.Step_01_Instructor_Logs_in_to_the_Application" finished-at="2015-10-06T16:14:32Z">
          <reporter-output>
            <line>
              <![CDATA[[ASSERT PASS]: element pageTitle with text Instructor Resource Center is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element courseOperation with text Manage Course is displayed.]]>
            </line>
            <line>
              <![CDATA[[ASSERT PASS]: element courseLink with text Mayank Test Book-Automated is displayed.]]>
            </line>
          </reporter-output>
        </test-method> <!-- Step_02_Instructor_Manage_Course -->
        <test-method status="PASS" signature="Step_03_Instructor_Drops_Student()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="Step_03_Instructor_Drops_Student" duration-ms="30948" started-at="2015-10-06T16:14:32Z" depends-on-methods="com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course.Step_02_Instructor_Manage_Course" finished-at="2015-10-06T16:15:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_03_Instructor_Drops_Student -->
        <test-method status="PASS" signature="takeScreenshotonFailure(org.testng.ITestResult)[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="takeScreenshotonFailure" is-config="true" duration-ms="0" started-at="2015-10-06T16:14:32Z" finished-at="2015-10-06T16:14:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Step_02_Instructor_Manage_Course status=SUCCESS method=NG_30181_TC03_Instructor_Deletes_Course.Step_02_Instructor_Manage_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] output=[ASSERT PASS]: element pageTitle with text Instructor Resource Center is displayed.]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotonFailure -->
        <test-method status="PASS" signature="takeScreenshotonFailure(org.testng.ITestResult)[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="takeScreenshotonFailure" is-config="true" duration-ms="2" started-at="2015-10-06T16:15:03Z" finished-at="2015-10-06T16:15:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Step_03_Instructor_Drops_Student status=SUCCESS method=NG_30181_TC03_Instructor_Deletes_Course.Step_03_Instructor_Drops_Student()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotonFailure -->
        <test-method status="FAIL" signature="Step_04_user_LogOut()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="Step_04_user_LogOut" duration-ms="30090" started-at="2015-10-06T16:15:03Z" depends-on-methods="com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course.Step_03_Instructor_Drops_Student" finished-at="2015-10-06T16:15:33Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.qait.mindtap.keywords.NG_30104.NG_30181_LoginPageActions.logOut(NG_30181_LoginPageActions.java:52)
	at com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course.Step_04_user_LogOut(NG_30181_TC03_Instructor_Deletes_Course.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[[ASSERT FAIL]: Element helloUser not found on the Header !!!]]>
            </line>
          </reporter-output>
        </test-method> <!-- Step_04_user_LogOut -->
        <test-method status="PASS" signature="takeScreenshotonFailure(org.testng.ITestResult)[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="takeScreenshotonFailure" is-config="true" duration-ms="2" started-at="2015-10-06T16:15:33Z" finished-at="2015-10-06T16:15:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Step_04_user_LogOut status=FAILURE method=NG_30181_TC03_Instructor_Deletes_Course.Step_04_user_LogOut()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] output=[ASSERT FAIL]: Element helloUser not found on the Header !!!]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotonFailure -->
        <test-method status="SKIP" signature="Step_06_Instructor_Manage_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="Step_06_Instructor_Manage_Course" duration-ms="0" started-at="2015-10-06T16:15:33Z" depends-on-methods="com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course.Step_05_Instructor_Logs_in_to_the_Application_Again" finished-at="2015-10-06T16:15:33Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method NG_30181_TC03_Instructor_Deletes_Course.Step_06_Instructor_Manage_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method NG_30181_TC03_Instructor_Deletes_Course.Step_06_Instructor_Manage_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_06_Instructor_Manage_Course -->
        <test-method status="SKIP" signature="Step_05_Instructor_Logs_in_to_the_Application_Again()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="Step_05_Instructor_Logs_in_to_the_Application_Again" duration-ms="0" started-at="2015-10-06T16:15:33Z" depends-on-methods="com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course.Step_04_user_LogOut" finished-at="2015-10-06T16:15:33Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method NG_30181_TC03_Instructor_Deletes_Course.Step_05_Instructor_Logs_in_to_the_Application_Again()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method NG_30181_TC03_Instructor_Deletes_Course.Step_05_Instructor_Logs_in_to_the_Application_Again()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_05_Instructor_Logs_in_to_the_Application_Again -->
        <test-method status="SKIP" signature="Step_07_Instructor_Delete_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="Step_07_Instructor_Delete_Course" duration-ms="0" started-at="2015-10-06T16:15:33Z" depends-on-methods="com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course.Step_06_Instructor_Manage_Course" finished-at="2015-10-06T16:15:33Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method NG_30181_TC03_Instructor_Deletes_Course.Step_07_Instructor_Delete_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method NG_30181_TC03_Instructor_Deletes_Course.Step_07_Instructor_Delete_Course()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_07_Instructor_Delete_Course -->
        <test-method status="PASS" signature="stop_test_session()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="stop_test_session" is-config="true" duration-ms="115" started-at="2015-10-06T16:15:33Z" finished-at="2015-10-06T16:15:33Z">
          <reporter-output>
            <line>
              <![CDATA[[INFO]: The Test: NG_30181_TC04_INSTRUCTOR_DELETES_COURSE COMPLETED!
]]>
            </line>
          </reporter-output>
        </test-method> <!-- stop_test_session -->
        <test-method status="SKIP" signature="Step_07_Instructor_SignOut()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5]" name="Step_07_Instructor_SignOut" duration-ms="0" started-at="2015-10-06T16:15:33Z" depends-on-methods="com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course.Step_07_Instructor_Delete_Course" finished-at="2015-10-06T16:15:33Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method NG_30181_TC03_Instructor_Deletes_Course.Step_07_Instructor_SignOut()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method NG_30181_TC03_Instructor_Deletes_Course.Step_07_Instructor_SignOut()[pri:0, instance:com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course@1be98f5] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:128)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Step_07_Instructor_SignOut -->
      </class> <!-- com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course -->
    </test> <!-- Command line test 2a15b366-bb28-4cd9-be7a-e20e75b8651d -->
  </suite> <!-- com.qait.mindtap.tests.NG_30104.NG_30181_TC03_Instructor_Deletes_Course -->
</testng-results>
